frontend/src/routes/__root.tsx

```diff
@@ -4,18 +4,142 @@ import { Logo } from "~/components/logo";
 import { MetaTitle } from "~/components/meta-title";
 import { TailwindIndicator } from "~/components/tailwind-indicator";
 import { Button } from "~/components/ui/button";
+import {
+  Menubar,
+  MenubarContent,
+  MenubarItem,
+  MenubarMenu,
+  MenubarTrigger
+} from "~/components/ui/menubar";
+
+import {
+  AlarmCheck,
+  BookOpen,
+  ChevronDown,
+  ChevronsUpDown,
+  CircleUser,
+  Folder,
+  Globe,
+  Hammer,
+  HeartHandshake,
+  HelpCircle,
+  LogOut,
+  Search,
+  Send,
+  Settings,
+  Twitter
+} from "lucide-react";
+import { useAuthUser } from "~/components/helper/use-auth-user";
+import { Input } from "~/components/ui/input";
 
 export const Route = createRootRoute({
   component: () => (
     <main className="bg-background">
+      <Navigation />
       <Outlet />
-      <TailwindIndicator />
-      <TanStackRouterDevtools />
+      <Footer />
```

Comment:
The html for the layout should be :

```tsx
<>
 <Header />
 <main> 
    <Outlet />
 </main>
 <Footer />
</>
```

The main content is distinct from the header and footer.